source/use-cases/sensitive-data-encryption.txt

```diff
@@ -166,7 +166,7 @@ Create JSON objects for each field and append them to the `properties` map.
 
 
 - View the full JSON Schema for the Medco Medical Management System `here <https://raw.githubusercontent.com/mongodb/docs-assets/DOCSP-json-schema-helper-and-json/MedcoMDSchema.json>`_
-- View the `helper code <https://raw.githubusercontent.com/mongodb/docs-assets/DOCSP-json-schema-helper-and-json/FieldLevelEncryptionJSONSchema.java>`_ to generate the full JSON Schema for this MedcoMD guide in Java. 
+- View the `helper code <https://raw.githubusercontent.com/mongodb/docs-assets/DOCSP-json-schema-helper-and-json/FieldLevelEncryptionJSONSchema.java>`_ to generate the full JSON Schema for this MedcoMD guide in Java
```

Comment:
I have been using a period to close out bullet points that are complete sentences, similar to how it is done here: 
https://docs.mongodb.com/ecosystem/drivers/specs/

What do you think about following that pattern (also followed this in my CSFLE PRs)?

Reply:
Generally context dependent on something like this. The most important thing is to be internally consistent in the list, i.e. if one ends with a period they all should.